### PR TITLE
feat: Added Support for Django 4.2

### PR DESCRIPTION
**Pull request by zubairshakoorarbisoft**
Issue: https://github.com/edx/upgrades/issues/128
Added support for Django 4.2

### CODE REVIEW

File: edx_credentials_themes/__init__.py [hunk 1]
```diff
@@ -1 +1,2 @@
-__version__ = '0.3.3'
+__version__ = '1.2.0'
```
Comment (justinhynes) [inline edx_credentials_themes/__init__.py:1]: maybe this should be `0.4.0` and not `1.x.x`?

**Comment by justinhynes** [comment]
Hi @UsamaSadiq, 

Is this upgrade still needed? It doesn't look like it's been updated in awhile and has some conflicts.

File: edx_credentials_themes/__init__.py (hunk 1 above)
Comment (UsamaSadiq) [reply edx_credentials_themes/__init__.py]: Updated to match the suggested `0.4.0`.

**Review by justinhynes — APPROVED** [review]

**Comment by justinhynes** [comment]
rebased and fixed a very minor file conflict so I can merge this change. thanks!

**Pull request merged**